components/common-data-items.js

```diff
@@ -164,18 +169,28 @@ export function SampleDataItems({ item, sources = null, children }) {
           </DataItemValue>
         </>
       )}
-      {item.sorted_fraction && (
+      {item.sorted_from && (
         <>
-          <DataItemLabel>Sorted Fraction</DataItemLabel>
+          <DataItemLabel>Sorted From Sample(s)</DataItemLabel>
           <DataItemValue>
-            <Link href={item.sorted_fraction["@id"]}>
-              {item.sorted_fraction.accession}
+            <Link href={item.sorted_from["@id"]}>
+              {item.sorted_from.accession}
             </Link>
-            {item.sorted_fraction_detail ? (
-              <> {item.sorted_fraction_detail}</>
-            ) : (
-              ""
-            )}
+            {item.sorted_from_detail ? <> {item.sorted_from_detail}</> : ""}
+          </DataItemValue>
+        </>
+      )}
+      {sortedFractions?.length > 0 && (
+        <>
+          <DataItemLabel>Sorted Fractions of Sample(s)</DataItemLabel>
```

Comment:
Another option, if you’re up for it, is to use the singular “Sample” if `sortedFractions.length === 1`, but “Samples” otherwise.

Reply:
done